examples/BucketExists.cc

```diff
@@ -15,7 +15,7 @@
 
 #include "client.h"
 
-int main(int argc, char* argv[]) {
+int main(int /*argc*/, char* /*argv*/[]) {
```

Comment:
Sure, that's a good idea.

Reply:
Or we can just silent the warning by `(void)argc`, etc...

Or even better:

```c++
template<typename... Args>
static inline void unused(Args&&...) noexcept {}
```

Then:

```c++
unused(argc, argv);
```

Maybe explicit is better than implicit? Because this could be used in other places as well (of course residing in a proper namespace).

Reply:
Fortunately, the short `main()` signature is legal, so I'll go with that option now.